gffread.wdl

```diff
@@ -0,0 +1,42 @@
+version 1.0
+
+task GffRead {
+    input {
+        String? preCommand
+        File inputGff
+        File genomicSequence
+        File? genomicIndex  # Optional. GFFRead can create this by itself.
+        String? exonsFastaPath
+        String? CDSFastaPath
+        String? proteinFastaPath
+        String? filteredGffPath
+        Boolean outputGtfFormat = false
+    }
+
+    # The mkdirs below are hackish. it should be
+    # ~{"mkir -p $(dirname " + somePath + ")"}
+    # but this goes wrong. Cromwell will always use ')' even if somepath is not defined. Which leads to crashing.
+    command {
+        set -e -o pipefail
+        ~{preCommand}
+        ~{"mkdir -p $(dirname " + CDSFastaPath}~{true=")" false="" defined(CDSFastaPath)}
```

Comment:
An alternative might be `mkdir -p $(dirname . ~{CDSFastaPath})`

If CDSFastaPath is not defined it will only try to create the current working directory, which already exists, so it won't do anything.